Esterni/ManualeUtente/components/c3-Installazione-MU.tex

```diff
@@ -1,36 +1,28 @@
 \section{Installazione ed avvio di SSD}
-Per installare ed avviare l'applicativo \gloman{SSD} bisogna effettuare i seguenti passaggi.
-\subsection{Scaricare il programma e le librerie}
+Per installare ed avviare l'applicazione \gloman{SSD} bisogna effettuare passi diversi in base al sistema operativo. Le varie versioni possono essere scaricate dal link: \newline{}
+\centerline{\url{https://github.com/MercurySeven/project-SSD/releases}}
 
+\subsection{Windows}
 \begin{itemize}
-	\item \textbf{Scaricare}: tutto il programma è presente nel seguente \gloman{Repository}:\newline{}
-\centerline{\url{https://github.com/MercurySeven/project-SSD}}
+\item \textbf{Scaricare:} cliccare sull'apposito link e scaricare il file denominato \textit{SSD.exe};
+\item \textbf{Avviare l'applicazione:} cliccare sull'applicazione appena scaricata.
+\end{itemize}
 
-	\item \textbf{Estrarre il contenuto del file .zip}: In una cartella a piacimento, che sia al di fuori della cartella che si vuole sincronizzare, utilizzare l'applicativo del computer per estrarre l'archivio nella cartella desiderata (7zip, WinRAR, etc);
-	\item \textbf{Installare le librerie richieste}:
-	Tramite terminale, posizionarsi sulla cartella dove è stato estratto il contenuto ed eseguire il seguente comando:\newline{}
-	\centerline{\texttt{pip install -r "requirements.txt"}}
+\subsection{MacOS}
+\begin{itemize}
+\item \textbf{Scaricare:} cliccare sull'apposito link e scaricare il file denominato \textit{SSD\_macOS.zip};
+\item \textbf{Avviare l'applicazione:} decomprimere il file, aprire la cartella, dare i permessi di esecuzione allo script launch.sh e avviarlo da terminale.
 \end{itemize}
-\subsection{Impostare PYTHONPATH}
-Per impostare il PYTHONPATH bisogna utilizzare la keyword set su Windows e la keyword export su macOS e Linux. La variabile deve essere impostata al percorso assoluto dove si è estratto l'archivio.
-Di seguito sono riportati due esempi, rispettivamente per Windows e Linux/macOS
-\begin{figure}[H]
-    \centering
-    \includegraphics[scale = 0.65]{components/img/Windows-istruzione-1.png}
-    \caption{Comando per impostare PYTHONPATH su Windows}
-    \label{fig:comando per impostare PYTHONPATH su windows}
-\end{figure}
 \begin{figure}[H]
     \centering
-    \includegraphics[scale = 0.50]{components/img/linux-istruzione-1.jpg}
-    \caption{ Comando per impostare PYTHONPATH su Linux/macOS}
-    \label{fig:comando per impostare PYTHONPATH su windows}
+    \includegraphics[scale = 0.3]{components/img/permessi-eseguire-ed-eseguire-launch.png}
+    \caption{Dare i permessi ed eseguire launch.sh}
+    \label{fig:Dare i permessi ed eseguire launch.sh}
```

Comment:
cambierei caption e label in: Avvio applicazione

Reply:
Ma non è l'avvio, i permessi devi darli solo al primo avvio ad esempio. E l'avvio applicazione non andrebbe bene comunque perchè dovresti specificare anche l'os (macos) altrimenti nell'indice uno legge solo avvio applicazione